fast-accountlookup-service/src/services/mockDataGenerator.ts

```
@@ -0,0 +1,186 @@
+    const accountGroup = AccountUtils.getAccountGroup(accountCategory);
+    const branchId = AccountUtils.generateBranchId();
+    
+    const physicalAcctInfo: any = {
+      acctId: cdtrAcctId,
+      acctSys: accountSystem,
+      acctGroup: accountGroup,
+      country: SINGAPORE_BANKING_CONSTANTS.COUNTRY_CODE,
+      acctAttributes: {
+        acctType: accountType,
+        acctCategory: accountCategory,
+        acctPurpose: this.generateAccountPurpose(accountType)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)